Admin/AbstractAdmin.php

```diff
@@ -1490,13 +1509,28 @@ public function getLabel()
 
     /**
      * @param bool $persist
+     *
+     * @deprecated since 3.x, to be removed in 4.0.
      */
     public function setPersistFilters($persist)
     {
+        @trigger_error(
+            'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in 4.0.',
+            E_USER_DEPRECATED
+        );
+
         $this->persistFilters = $persist;
```

Comment:
I'm not sure to understand what you want for this method.

Are asking me to 

- set `$this->filterPersister` to `null` if `$persist` = `false`
- set `$this->filterPersister` to `new SessionFilterPersister()` if `$persist` = `true`

?